<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/html" xml:lang="en">
  <head>
    <meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" href="jemdoc.css" type="text/css" />
    <link rel="shortcut icon" type="image/x-icon" href="figs/favicon.ico">
    <link rel="bookmark" type="image/x-icon" href="figs/favicon.ico"/>
    <title>Yanjun Li</title>
  </head>

  <body>
    <table summary="Table for page layout." id="tlayout">
      <tr valign="top">
      <td id="layout-menu">
        <div class="menu-category">Yanjun Li</div>
          <div class="menu-item"><a href="index.html" class="current">Home</a></div>
          <div class="menu-item"><a href="bio.html">Biography</a></div>
          <div class="menu-item"><a href="https://scholar.google.com/citations?user=bUvL4BEAAAAJ&hl=en">Google Scholar</a></div>
          <div class="menu-item"><a href="https://github.com/YanjunLi-CS">GitHub</a></div>
        <div class="menu-category">Research</div>
<!--          <div class="menu-item"><a href="publication.com/">Publication list</a></div>-->
          <div class="menu-item"><a href="useful_link.html">Useful Links</a></div>
          <div class="menu-item"><a href="contact.html">Contact</a></div>
      </td>
      <td id="layout-content">
        <div id="toptitle">
          <h1>Yanjun Li</h1>
        </div>
        <table class="imgtable">
          <tr>
            <td><a href="https://yanjunlics.github.io/"><img src="figs/bio.jpg" alt="alt text" width="196.5px" height="204px" /></a>&nbsp;
            </td>
            <td align="left">
              <p><b>Yanjun Li</b></p>
              <p>Machine Learning Scientist<br />
              <a href="https://www.calicolabs.com/">Calico Life Sciences</a><br />
              1170 Veterans Blvd <br />
              South San Francisco, CA, 94080, U.S. <br /> </p>
              <p>E-mail: <a href="mailto:yanjun.li.cs@gmail.com">yanjun.li.cs@gmail.com</a><br /> </p>
              <a href="https://scholar.google.com/citations?user=bUvL4BEAAAAJ&hl=en">Google Scholar,</a>
              <a href="https://github.com/YanjunLi-CS">GitHub,</a>
              <a href="https://twitter.com/YanjunLi_CS">Twitter</a>
              </p>
            </td>
          </tr>
      </table>
      <h2>Biography</h2>
        <p>
        Dr. Yanjun Li is currently a Machine Learning Scientist at Calico Life Sciences,
          an <a href="https://en.wikipedia.org/wiki/Alphabet_Inc.#:~:text=Executive%20Leader-,Calico,Arthur%20D.%20Levinson,-CapitalG">Alphabet-founded</a>
          research and development lab for human aging. His research interest spans the fields of deep learning, AI-driven drug discovery,
          structural genomics and proteomics, and precision medicine.
        Li's work aims to develop innovative AI algorithms to tackle the foundational life science challenges with a broad scientific impact,
        and optimize and automatize the real-world drug discovery and design pipeline.
        </p>
        Dr. Li was a senior research scientist at the Institute of Deep Learning, Baidu Research USA,
        Sunnyvale, California.
        He received his Ph.D. in computer science from University of Florida (2021),
        advised by <a href="https://www.cs.cityu.edu.hk/~dapengwu/">Dr. Dapeng Oliver Wu</a> and
        <a href="https://cognization.com/andyli/">Dr. Xiaolin (Andy) Li.</a>
        He received his dual M.E. from University of Electronic Science and Technology of China and Waseda University (2015)
        and B.E. from University of Electronic Science and Technology of China (2012).
      <h2>Research Interests</h2>
        <ul>
          <li><p>AI-driven Drug Discovery</p>
            <ul><li>Small Molecule Design, Small Molecule Property Prediction, Protein-ligand Binding Affinity Prediction, Drug-drug Interaction Prediction, etc</li></ul>
          </li>
          <li><p>Structural Genomics and Proteomics</p>
            <ul><li>Deep Learning for Peptide/Antibody/General Protein Fix-backbone and <i>de novo</i> Design, Proteinâ€“protein Interaction Prediction, Genetic Mutations Effect Prediction, etc</li></ul>
          </li>
          <li><p>Precision Medicine</p>
            <ul><li>Single-Cell RNA-sequencing, Spatial transcriptomics, Drug Repurposing, Drug Synergy and Combination, etc</li></ul>
          </li>
        </ul>
      <h2>Education</h2>
       <ul>
         <li><p>Ph. D. in Computer Science, University of Florida, Gainesville, FL, USA, 2021</p></li>
         <li><p>M. E. in Communication and Information System, University of Electronic Science and Technology of China, Chengdu, China, 2015</p></li>
         <li><p>M. E. in Information, Production and Systems Engineering, Waseda University, Kitakyushu, Japan, 2015</p></li>
         <li><p>B. E. in Information Engineering, University of Electronic Science and Technology of China, Chengdu, China, 2012</p></li>
       </ul>

      <h2>Professional Experience</h2>
        <ul>
          <li><p>Machine Learning Scientist, Calico Life Sciences, 09.2022-Present</p></li>
          <li><p>Senior Research Scientist, Institute of Deep Learning USA, Baidu Research USA, 06.2021-08.2022</p></li>
          <li><p>AI/ML Ph.D. Intern, GE Healthcare, 06.2020-08.2020</p></li>
        </ul>

      <h2>Selected Publications</h2>
        <p>* indicates the equal contribution. <a href="https://scholar.google.com/citations?user=bUvL4BEAAAAJ&hl=en">(Full publication list)</a></p>
        <ol>
          <li><p>
            <b>Y. Li</b>, D. Zhou, G. Zheng, D. Wu, X. Li, Y. Yuan, DyScore: A Boosting Scoring Method with Dynamic Properties for Identifying True Binders and Non-binders in Structure-based Drug Discovery.
            <i>Journal of Chemical Information and Modeling</i> (IF = 6.162), 2022.
            [<a href="https://pubs.acs.org/doi/10.1021/acs.jcim.2c00926#:~:text=NEXT-,DyScore%3A%20A%20Boosting%20Scoring%20Method%20with%20Dynamic%20Properties%20for%20Identifying,in%20Structure%2DBased%20Drug%20Discovery">doi</a>]
            [<a href="https://github.com/salemilab/DeepDynaTree">code</a>]
            [<a href="http://dyscore.deepdrug.net/">web server</a>]
          </p></li>
          <li><p>
            C. Sun*, <b>Y. Li</b>*, S. Marini, D. Wu, M. Salemi, B. Magalis,
            Phylogenetic-informed graph deep learning to classify dynamic transmission clusters in infectious disease epidemics, 2022.
            [<a href="https://www.biorxiv.org/content/10.1101/2022.04.10.487587v2">arXiv</a>]
            [<a href="https://github.com/YanjunLi-CS/dyscore">code</a>]
          </p></li>
          <li><p>
            B. Liu*, <b>Y. Li</b>*, L. Zhang, Analysis and visualization of spatial transcriptomic data.
            <i>Frontiers in Genetics</i>, 2022.
            [<a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.785290/full">doi</a>]</p>
          </p></li>
          <li><p>
            Y. Ren, T. Loftus, <b>Y. Li</b>, Z. Guan, M. Ruppert, S. Datta, G. Upchurch, P. Tighe, P. Rashidi, B. Shickel, T. Baslanti, A. Bihorac,
            Physiologic signatures within six hours of hospitalization identify acute illness phenotypes. <i>Plos Digital Health</i>, 2022.
            [<a href="https://journals.plos.org/digitalhealth/article?id=10.1371/journal.pdig.0000110">doi</a>]
          </p></li>
          <li><p>
            <b>Y. Li</b>, C. Sun, D. Romanova, D. Wu, L. Moroz,
            Book Chapter: Analysis and visualization of Mnemiopsis Leidyi single-cell data with Scanpy and MetaCell: a tutorial.
            <i>Methods in Molecular Biology, Springer</i>, 2022.
          </p></li>
          <li><p>
            S. Datta, <b>Y. Li</b>, M. Ruppert, Y. Ren, B. Shickel, T. Baslanti, P. Rashidi, A. Bihorac,
            Reinforcement learning in surgery. <i>Surgery</i>, 2021
            [<a href="https://www.sciencedirect.com/science/article/pii/S0039606020308254">doi</a>]
          </p></li>
          <li><p>
            M. Rezaei*, <b>Y. Li</b>*, D. Wu, X. Li,  C. Li, Deep Learning in Drug Design: Protein-Ligand Binding Affinity Prediction.
            <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>, 2020.
            [<a href="https://ieeexplore.ieee.org/document/9305294">doi</a>]
          </p></li>
          <li><p>
            <b>Y. Li</b>, S. Yu, J. Principe, X. Li, D. Wu,
            PRI-VAE: Principle of Relevant Information Variational Autoencoders. 2020.
            [<a href="https://arxiv.org/abs/2007.06503">arXiv</a>]
          </p></li>
          <li><p>
            T. Loftus, A. Filiberto, <b>Y. Li</b>, J. Balch, P. Tighe, G. Upchurch, P. Rashidi, X. Li, A. Bihorac,
            Decision Analysis and Reinforcement Learning in Surgical Decision-Making.
            <i>Surgery</i>, 2020
            [<a href="https://www.sciencedirect.com/science/article/abs/pii/S0039606020302610">doi</a>]
          </p></li>
          <li><p>
            <b>Y. Li</b>, M. Rezaei, C. Li, X. Li, DeepAtom: A Framework for Protein-Ligand Binding Affinity Prediction.
            <i>Intl. Conf. on Bioinformatics and Biomedicine (BIBM)</i>, 2019. (acceptance rate 18%)
            [<a href="https://www.computer.org/csdl/proceedings-article/bibm/2019/08982964/1hgu0lnu98c">doi</a>]
          </p></li>
          <li><p>
            <b>Y. Li</b>, K. Heng, K. Ye, X. Li, FoldingZero: Protein Folding from Scratch in Hydrophobic-Polar Model.
            <i>Deep Reinforcement Learning Workshop (Oral) of NeurIPS</i>, 2018. (acceptance rate <10%)
              [<a href="https://arxiv.org/abs/1812.00967">arXiv</a>]
          </p></li>
          <li><p>
            L. Sundaram, H. Gao, S. Padigepati, J. McRae, <b>Y. Li</b>, et al.
<!--            J. Kosmicki, N. Fritzilas, J. Hakenberg, A. Dutta, J. Shon, J. Xu, S. Batzoglou, X. Li, K. Farh.-->
            Predicting the clinical impact of human mutation with deep neural networks.
            <i>Nature Genetics</i> (IF = 41.38)
            [<a href="https://www.nature.com/articles/s41588-018-0167-z">doi</a>]
            [<a href="https://github.com/Illumina/PrimateAI">code</a>]
          </p></li>
          <li><p>
            Q. Zhu, <b>Y. Li</b>, X. Li. Character Sequence-to-Sequence Model with Global Attention for Universal Morphological Reinflection.
            <i>Proceedings of the CoNLL SIGMORPHON 2017 Shared Task: Universal Morphological Reinflection</i>, 2017.
            [<a href="https://aclanthology.org/K17-2009/">doi</a>]
          </p></li>
          <li><p>
            <b>Y. Li</b>, X. Li, O. Yoshie.
            Traffic Engineering Framework with Machine Learning based Meta-layer in Software-defined Networks.
            <i>Intl. Conf. on Network Infrastructure and Digital Content (IC-NIDC)</i>, 2014.
            [<a href="https://ieeexplore.ieee.org/document/7000278">doi</a>]
          </p></li>
        </ol>

      <h2>Awards</h2>
        <ul>
        <li><p>Student Travel Grant, IEEE Intl. Conf. on Bioinformatics and Biomedicine, 2019</p>
        </li>
        <li><p>Mentorship Excellence Award, Summer Deep Learning Workshop, University of Florida, 2018</p>
        </li>
        <li><p>Honors Scholarship for International Students, Waseda University, 2014</p>
        </li>
        <li><p>Scholarship for Advancement of Industry, Science and Technology, Kitakyushu, Japan, 2014</p>
        </li>
        <li><p>Outstanding Graduates Awards, University of Electronic Science and Technology of China, 2012</p>
        </li>
        <li><p>Special-class Scholarship, University of Electronic Science and Technology of China, 2011</p>
        </li>
        </ul>

      </td>
      </tr>
    </table>
  </body>
</html>